src/Framework/PaymentGateways/Traits/ResponseHelpers.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Give\Framework\PaymentGateways\Traits;
+
+use Give\Framework\Http\Response\Types\JsonResponse;
+use Give\Framework\Http\Response\Types\RedirectResponse;
+
+trait ResponseHelpers {
```

Comment:
@jonwaldstein Let's change this to `HandleHttpResponses` — the "helpers" term is easily overused and not really descriptive. It hangs out playing buzzed darts with its cousin Utility. 😆

Reply:
Fair enough lol dd9ea2cd